tests/std/tests/P2442R1_views_chunk/test.cpp

```diff
@@ -197,12 +197,13 @@ constexpr bool test_one(Rng&& rng, Expected&& expected) {
         }
     }
 
-    STATIC_ASSERT(CanMemberEnd<const R> == forward_range<const V>);
+    STATIC_ASSERT(CanMemberEnd<const R> == input_range<const V>);
     if constexpr (CanMemberEnd<const R>) {
         const same_as<sentinel_t<const R>> auto cs = as_const(r).end();
         assert((r.begin() == cs) == is_empty);
         STATIC_ASSERT(
-            common_range<R> == (common_range<const V> && (sized_range<const V> || !bidirectional_range<const V>) ));
+            common_range<
+                R> == (forward_range<const V> && common_range<const V> && (sized_range<const V> || !bidirectional_range<const V>) ));

```

Comment:
You could try to force clang-formats hand by adding a comment
```suggestion
            common_range<R> //
                == (forward_range<const V> && common_range<const V> && (sized_range<const V> || !bidirectional_range<const V>) ));
```